### DISCUSSION TITLE
spark streaming写入doris问题

### DISCUSSION THREAD

**Discussion by jwsmai** [apache/doris#17426] [RFC]
<img width="945" alt="image" src="https://user-images.githubusercontent.com/63184217/222948412-2162e4a8-b6bb-49d9-a194-d1858a44bfc4.png">
非常奇怪的问题，普通的字符串写入时被当成了非法的json。

Apache Spark 3.2.1, Scala 2.12, Doris version 0.15.0-release-c9b27f0
相关包：org.apache.doris:spark-doris-connector-3.1_2.12:1.1.0

**Comment by LemonLiTree** [comment]
doris升级看看？
